superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -154,6 +154,77 @@ export const visTypes = {
     ],
   },
 
+  line_xy: {
+    label: t('XY - Line Chart'),
+    controlPanelSections: [
+      {
+        label: t('Axis Options'),
+        description: t('Select which columns or metrics to plot'),
+        expanded: true,
+        controlSetRows: [
+          ['columns_and_metrics_x'],
+          ['columns_and_metrics_y'],
+          ['slice_by'],
+        ],
+      },
+      {
+        label: t('GROUP BY'),
+        description: t('Use this section if you want a query that aggregates'),
+        controlSetRows: [
+          ['groupby'],
+          ['metrics'],
+          ['limit', 'timeseries_limit_metric'],
+          ['order_desc', null],
+        ],
+      },
+      {
+        label: t('NOT GROUPED BY'),
```

Comment:
Handling `NOT GROUPED BY` here adds a lot of complexity to this visualization, why is it necessary?

Reply:
Just to note - I included `NOT GROUPED BY` to address the feature @rumbin mentioned above.